### PR TITLE
Update fibonacci.hpp

### PR DESCRIPTION
**Pull request by KamilFinanseOrazProgramowanie**
Solution

### CODE REVIEW

File: module1/homework/fibonacci/fibonacci.hpp [hunk 1]
```diff
@@ -1,11 +1,24 @@
...
+    return b;
 }
 
 int fibonacci_recursive(int sequence) {
-    // TODO: Your implementation goes here
-    return 0;
+    if (sequence == 0 || sequence == 1){
+        return sequence;
+        }
+    return (fibonacci_recursive(sequence - 1) + fibonacci_recursive(int sequence - 2));
```
Comment (TomaszEichler) [inline module1/homework/fibonacci/fibonacci.hpp:23]: delete `int` in your last line of code. 
`return (fibonacci_recursive(sequence - 1) + fibonacci_recursive(sequence - 2));
 `
 should work

File: module1/homework/fibonacci/fibonacci.hpp [hunk 2]
```diff
@@ -1,11 +1,24 @@
 #pragma once
 
 int fibonacci_iterative(int sequence) {
-    // TODO: Your implementation goes here
-    return 0;
+    
+    if( (sequence == 0) || (sequence == 1) ){
+    return sequence;
```
Comment (piezol) [inline module1/homework/fibonacci/fibonacci.hpp:6]:
```suggestion
        return sequence;
```
Indendation

File: module1/homework/fibonacci/fibonacci.hpp [hunk 3]
```diff
@@ -1,11 +1,24 @@
...
+    }
+    return b;
 }
 
 int fibonacci_recursive(int sequence) {
-    // TODO: Your implementation goes here
-    return 0;
+    if (sequence == 0 || sequence == 1){
+        return sequence;
+        }
```
Comment (piezol) [inline module1/homework/fibonacci/fibonacci.hpp:22]:
```suggestion
    if (sequence == 0 || sequence == 1) {
        return sequence;
    }
```
Formatting, indentation

**Comment by ziobron** [comment]
{ "requirement_ids": [1119, 1430], "points": [2, 6] }

**Pull request closed without merging**